src/App.js

```diff
@@ -1,29 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <section>
+    <Calculator />
+  </section>
+);
```

Comment:
- [ ] The `App.js` component should be class-based as per the project's requirement. Kindly refactor it to the class-based component. :pray: